libs/components/indicators/src/lib/modules/key-info/key-info-layout-type.ts

```diff
@@ -0,0 +1 @@
+export type SkyKeyInfoLayoutType = 'horizontal' | 'vertical';
```

Comment:
Needs a JSDocs description.

Reply:
Should union types get docs comments? None of the other ones I've looked at have one

Reply:
We have some inconsistency in our existing source code, but any type that gets added to our documentation should have a description. Here's an example from Affix: https://github.com/blackbaud/skyux/blob/main/libs/components/core/src/lib/modules/affix/affix-horizontal-alignment.ts